files/default/slack_handler_webhook.rb

```diff
@@ -80,7 +80,12 @@ def slack_message(message, webhook, text_attachment = nil)
     http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
I'd be okay with adding it as an attribute. It feels like we should always check SSL; I didn't realize it was disabled (I didn't write much of this original code).